cmd/metal-api/internal/metal/partition.go

```diff
@@ -29,3 +36,62 @@ func (sz Partitions) ByID() PartitionMap {
 	}
 	return res
 }
+
+type ScalerRange struct {
+	WaitingPoolMinSize string
+	WaitingPoolMaxSize string
```

Comment:
Could be a little better to continue saving the min and max in the database and decouple the `ScalerRange` from the database. Advantage would be: We can have a constructor that prevents outer modification of the min/max values and the constructor could also do the parsing / validation. This way, we error out as early as possible and from that point we can be sure that everything will be alright with the outputs. 

Don't know if you like it: Changing the signature to `Min(of int)` and `Max(of int)`, the functions can also do the calculation the percentage value if necessary and just return the number?